### PR TITLE
Test integrate black

### PR DESCRIPTION
**Pull request by klahnakoski**
Remove the rest of `isort` mentions

### CODE REVIEW

**Comment by SuyashSalampuria** [comment]
Thanks a lot for it @klahnakoski

**Comment by klahnakoski** [comment]
@SuyashSalampuria   do not thank me yet, if this does not pass tests, then I give it back to you

Have a good day.

**Comment by SuyashSalampuria** [comment]
I will be glad to do it :)

**Comment by SuyashSalampuria** [comment]
Tests passed
Still if anything is left
I will do it @klahnakoski

**Comment by klahnakoski** [comment]
@armenzg since you are owner, we will require your approval.  This is @SuyashSalampuria work, with me chopping out the rest of the `isort`

File: README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 [![Node dependencies Status](https://david-dm.org/mozilla/treeherder/status.svg)](https://david-dm.org/mozilla/treeherder)
 [![Node devDependencies Status](https://david-dm.org/mozilla/treeherder/dev-status.svg)](https://david-dm.org/mozilla/treeherder?type=dev)
 [![Documentation Status](https://readthedocs.org/projects/treeherder/badge/?version=latest)](https://treeherder.readthedocs.io/?badge=latest)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```
Comment (armenzg) [inline README.md:8]: What are you trying to do? That's not pointing to Treeherder specific status like the other badges.

Comment (SuyashSalampuria) [reply README.md:8]: I had seen this in the repo of black
They asked to use this batch when we use black

Comment (SuyashSalampuria) [reply README.md:8]: https://github.com/psf/black#show-your-style

**Comment by klahnakoski** [comment]
@SuyashSalampuria thank you for this!  life will be better now:  No more whitespace fights!

**Pull request merged**